src/components/LoginPage/LoginPageComponent.js

```diff
@@ -1,67 +1,69 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import slide1 from 'components/LoginPage/images/slide-1.png';
-import slide2 from 'components/LoginPage/images/slide-2.png';
-import slide3 from 'components/LoginPage/images/slide-3.png';
-import slide4 from 'components/LoginPage/images/slide-4.png';
+import styled from 'styled-components';
+import Slide1 from 'components/LoginPage/Slide1';
+import Slide2 from 'components/LoginPage/Slide2';
+import Slide3 from 'components/LoginPage/Slide3';
+import Slide4 from 'components/LoginPage/Slide4';
+import Slide5 from 'components/LoginPage/Slide5';
+import iconGoogle from 'components/LoginPage/images/icon-google.svg';
+
+const Main = styled.main`
+  width: 100%;
+  background-color: #131c3e;
+  font-family: Roboto;
+`;
+
+const Article = styled.article`
+  width: 100%;
+  max-width: calc(100vh*1.1);
+  margin: auto;
+`;
+
+const H3 = styled.h3`
+  padding-bottom: 7%;
+  font-size: calc(13px + (24 - 13)*(100vw - 375px)/(1200 - 375));
```

Comment:
Please leave some hints on how to read the formula. For example add a comment with a link to the formula, so others can figure out what each number here mean.

Reply:
I found that used `vmin` units is more appropriative to our task. Here some description 
[Specification](https://www.w3.org/TR/css-values-4/#vmin)
[Vmin on WebReference](https://webref.ru/css/unit/vmin)

Done in commit [db8d5d2](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/db8d5d213e208ca30de77f1c6674c40053f31f87)